pkg/admin/repair.go

```diff
@@ -0,0 +1,218 @@
+				FormattedMessage: fmt.Sprintf("unable to get container parameters from daemonset; %v\n", err),
+			},
+		)
+		return nil, err
+	}
+
+	resultCh := client.NewDriveLister().
+		DriveIDSelector(args.DriveIDs).
+		IgnoreNotFound(true).
+		List(ctx)
+	for result := range resultCh {
+		if result.Err != nil {
+			return results, result.Err
+		}
+
+		jobName := "repair-" + result.Drive.Name
+		if _, err := k8s.KubeClient().BatchV1().Jobs(consts.AppName).Get(ctx, jobName, metav1.GetOptions{}); err != nil {
+			if !apierrors.IsNotFound(err) {
+				log(
+					LogMessage{
+						Type:             ErrorLogType,
+						Err:              err,
+						Message:          "unable to get repair job",
+						Values:           map[string]any{"jobName": jobName},
+						FormattedMessage: fmt.Sprintf("unable to get repair job %v; %v\n", jobName, err),
+					},
+				)
+				continue
+			}
+		} else {
+			log(
+				LogMessage{
+					Type:             ErrorLogType,
+					Err:              err,
+					Message:          "job already exists",
+					Values:           map[string]any{"jobName": jobName},
+					FormattedMessage: fmt.Sprintf("job %v already exists\n", jobName),
+				},
+			)
+			continue
+		}
+
+		nodeID := string(result.Drive.GetNodeID())
+
+		containerArgs := []string{"/directpv", "repair", result.Drive.Name, "--kube-node-name=" + nodeID}
+		if args.ForceFlag {
+			containerArgs = append(containerArgs, "--force")
+		}
+		if args.DisablePrefetchFlag {
+			containerArgs = append(containerArgs, "--disable-prefetch")
+		}
+		if args.DryRun {
+			containerArgs = append(containerArgs, "--dry-run")
+		}
+
+		backOffLimit := int32(1)
+
+		volumes := []corev1.Volume{
+			k8s.NewHostPathVolume(consts.AppRootDirVolumeName, consts.AppRootDirVolumePath),
+			k8s.NewHostPathVolume(consts.LegacyAppRootDirVolumeName, consts.LegacyAppRootDirVolumePath),
+			k8s.NewHostPathVolume(consts.SysDirVolumeName, consts.SysDirVolumePath),
+			k8s.NewHostPathVolume(consts.DevDirVolumeName, consts.DevDirVolumePath),
+			k8s.NewHostPathVolume(consts.RunUdevDataVolumeName, consts.RunUdevDataVolumePath),
+		}
+
+		volumeMounts := []corev1.VolumeMount{
+			k8s.NewVolumeMount(consts.AppRootDirVolumeName, consts.AppRootDirVolumePath, corev1.MountPropagationBidirectional, false),
+			k8s.NewVolumeMount(consts.LegacyAppRootDirVolumeName, consts.LegacyAppRootDirVolumePath, corev1.MountPropagationBidirectional, false),
+			k8s.NewVolumeMount(consts.SysDirVolumeName, consts.SysDirVolumePath, corev1.MountPropagationBidirectional, false),
+			k8s.NewVolumeMount(consts.DevDirVolumeName, consts.DevDirVolumePath, corev1.MountPropagationHostToContainer, true),
+			k8s.NewVolumeMount(consts.RunUdevDataVolumeName, consts.RunUdevDataVolumePath, corev1.MountPropagationBidirectional, true),
+		}
+
+		privileged := true
+
+		job := batchv1.Job{
+			ObjectMeta: metav1.ObjectMeta{
+				Name:      jobName,
+				Namespace: consts.AppName,
+			},
+			Spec: batchv1.JobSpec{
+				BackoffLimit: &backOffLimit,
```

Comment:
Can we add `.spec.ttlSecondsAfterFinished` to automatically clean the completed/failed jobs after n mins. This will also allow the user to retry after n mins.